modules/ding_loan/plugins/content_types/loans.inc

```diff
@@ -168,10 +176,13 @@ function ding_loan_loans_form($form, &$form_state, $account, $loans = array()) {
       // about the associated material.
       /** @var \TingEntity $entity */
       $entity = $loan->entity;
+
       if (!is_object($entity) || NULL === $entity->getTingObject()) {
         // Patron have this item in loans list, but the search provider can't
-        // find this record for current agencyId, so we load it separately.
-        $ting_object = ting_search_object_load($loan->ding_entity_id);
+        // find this record for current agencyId, so ask ding_provider for help:
```

Comment:
These changes I am uncertain about.

Reply:
Is this needed after code was changes from $entity to $ting_object?

Reply:
- Original change: https://github.com/easySuite/ding2/pull/423/files#diff-6023214c29e0026da2b604ef2d5703aeL183
- Changes that was done by others: https://github.com/ding2/ding2/commit/0448d72e686685c780de74f9957149a3cd273898
- Compare our 4.2.1 with 4.4.2: https://github.com/easySuite/ding2/compare/hotfix-4.2.1...development-4.4.x#diff-6023214c29e0026da2b604ef2d5703aeL170